regml.py

```diff
@@ -577,13 +577,13 @@ def json_through(cfr_title, cfr_part, through=None, suppress_output=False):
         return
 
     print(colored("\nApplying JSON through {0[0]}\n".format(
-                  regml_regs[last_ver_idx-1]),
+                  regml_regs[last_ver_idx]),
           attrs=['bold']))
 
     # Perform the json application process
     # Unlike apply-through, since json outputs its own command line output, here we
     # reuse the existing json structure
-    json_command(regulation_files[:last_ver_idx])
+    json_command(regulation_files[:last_ver_idx+1])
```

Comment:
last_ver_idx should account for the +1 portion of this statement since it's constructed in the if/else just above.